lib/tests/e2e/suite.go

```diff
@@ -86,3 +87,55 @@ func (s *E2ETestSuite) TestBankSendBroadcastTxWithFileLock() {
 	s.Require().NoError(err)
 	assert.Equal(s.T(), "[]", txResponse.RawLog)
 }
+
+func (s *E2ETestSuite) TestLoadKeys() {
+	s.T().SkipNow()
+	_, err := setKeys()
+	if err == nil {
+		s.Require().NoError(loadKeys())
+	}
+	s.Require().NoError(s.network.WaitForNextBlock())
+}
+
+func (s *E2ETestSuite) TestOccSigning() {
+	s.T().SkipNow()
+	val := s.network.Validators[0]
+
+	k, err := val.ClientCtx.Keyring.Key(sample.Name)
+	s.Require().NoError(err)
+
+	addr, err := k.GetAddress()
+	s.Require().NoError(err)
+
+	coin := sdk.NewCoins(sdk.NewInt64Coin("stake", 10))
+	msg := banktypes.NewMsgSend(addr, val.Address, coin)
```

Comment:
Can we get the address from the trust wallet and remove the dependency from the keyring here completely?